Src/FluentAssertions/Primitives/DateTimeAssertions.cs

```diff
@@ -157,10 +158,7 @@ public AndConstraint<TAssertions> Be(DateTime? expected, string because = "", pa
     public AndConstraint<TAssertions> BeCloseTo(DateTime nearbyTime, TimeSpan precision, string because = "",
         params object[] becauseArgs)
     {
-        if (precision < TimeSpan.Zero)
-        {
-            throw new ArgumentOutOfRangeException(nameof(precision), $"The value of {nameof(precision)} must be non-negative.");
-        }
+        Guard.ThrowIfArgumentIsNegative(precision, nameof(precision), "The value must be non-negative.");
```

Comment:
Have you considered making this `"The value must be non-negative."` text be the default for `ThrowIfArgumentIsNegative`?